manifest.toml

```diff
@@ -0,0 +1,209 @@
+#:schema https://raw.githubusercontent.com/YunoHost/apps/master/schemas/manifest.v2.schema.json
+
+packaging_format = 2
+
+id = "gitea"
+name = "Gitea"
+description.en = "Lightweight Git forge"
+description.fr = "Forge Git légère"
+
+version = "1.20.4~ynh2"
+
+maintainers = ["Josué Tille"]
+# previous_maintainers = [
+#     "rafi59"
+# ]
+
+[upstream]
+license = "MIT"
+website = "https://gitea.io/"
+admindoc = "https://docs.gitea.io/"
+code = "https://github.com/go-gitea/gitea"
+cpe = "cpe:2.3:a:gitea:gitea"
+fund = "https://opencollective.com/gitea"
+
+[integration]
+yunohost = ">= 11.2"
+architectures = [
+    "amd64",
+    "i386",
+    "armhf",
+    "arm64"
+]
+multi_instance = true
+ldap = true
+sso = false
+disk = "120M"
+ram.build = "100M"
+ram.runtime = "100M"
+
+[install]
+    [install.domain]
+    # this is a generic question - ask strings are automatically handled by Yunohost's core
+    type = "domain"
+
+    [install.path]
+    # this is a generic question - ask strings are automatically handled by Yunohost's core
+    type = "path"
+    default = "/gitea"
+
+    [install.admin]
+    # this is a generic question - ask strings are automatically handled by Yunohost's core
+    type = "user"
+
+    [install.init_main_permission]
+    help.en = "A public server means that everybody is able to access to the main page of the forge, on the public profile of the user and on the public repository. But you still can limit the access to each repository if you set it as private. Note that to be able to use the remote Git command (clone, pull, push) with HTTP and to use the API by (by example with a smartphone), you need to set this application as public."
```

Comment:
We need to update this to the question and choices provided : visitors, all users, admins

Reply:
This is fixed.